packages/stylelint-config/rules/stylelint-config-lmc.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  rules: {
+    /**
+     * Rules added from `old stylelint list` not included in any extension package:
+     */
+
+    // Reason: Taken over from LMC historical/preferred code style settings.
+    // Docs: https://stylelint.io/user-guide/rules/list/declaration-no-important
+    'declaration-no-important': [
+      true,
+      {
+        severity: 'warning',
+      },
+    ],
+
+    // Reason: Taken over from LMC historical/preferred code style settings.
+    // Docs: https://stylelint.io/user-guide/rules/list/selector-max-class
+    'selector-max-class': 5,
+
+    // Reason: Taken over from LMC historical/preferred code style settings.
+    // Docs: https://stylelint.io/user-guide/rules/list/selector-max-id
+    'selector-max-id': 1,
+
+    // Reason: Taken over from LMC historical/preferred code style settings.
+    // Docs: https://stylelint.io/user-guide/rules/list/font-weight-notation
+    'font-weight-notation': 'numeric',
+
+    // Reason: Taken over from LMC historical/preferred code style settings.
+    // Docs: https://stylelint.io/user-guide/rules/list/unit-allowed-list
+    'unit-allowed-list': ['em', 'rem', '%', 's', 'px', 'pt', 'ex', 'deg', 'cm', 'fr', 'vh', 'vw'],
+
+    // Reason: Taken over from LMC historical/preferred code style settings.
+    // Docs: https://stylelint.io/user-guide/rules/list/color-named
+    'color-named': 'never',
+
+    /**
+     * To discuss - rule modifications due to extension packages implementation - wasn't a problem before, now it is:
+     * THESE RULES IS GOOD TO BE KEPT ENABLED!
+     */
+    'scss/dollar-variable-pattern': null,
+    'scss/dollar-variable-empty-line-before': null,
+    'scss/at-mixin-argumentless-call-parentheses': null,
+    'scss/at-rule-conditional-no-parentheses': null,
+    'scss/operator-no-newline-after': null,
+    'scss/function-unquote-no-unquoted-strings-inside': null,
```

Comment:
In some cases it is handy to use functions like `unquote`.

Please give feedback whether you think it should be kept enabled as I do or not. Thank you.

Reply:
Conversation solved by internal discussion.